### PR TITLE
Mobile main page with social media connect

### PR DESCRIPTION
**Pull request by asharanjith**
In this, I am making the Mobile Main Page of my Portfolio.

- Add linter and proper documentation
- Add toolbar and assets folder to store all images
- Create a description of myself
- Create social media link
- Did styling for images and icons
- Run linter and fix validation errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <div class="toolbar">
+        <h3 class="logo">My logo</h3>
+        <i class="fas fa-bars"></i>
+    </div>
+        <div class="Ellipse11"><img src="\Assets\Ellipse 11.png"></div>
+        <div class="Ellipse3"><img src="\Assets\Ellipse 3.png"></div>
+        <div class="Rectangle4"><img src="\Assets\Rectangle 4.png"></div>
+        <div class="Rectangle7"><img src="\Assets\Rectangle 7.png"></div>
+        <div class="Ellipse5"><img src="\Assets\Ellipse 5.png"></div>
+        <div class="Ellipse8"><img src="\Assets\Ellipse 8.png"></div>
```
Comment (Santiago220991) [inline index.html]: Kindly fix the issue with the images they are not rendering. You could try editing the src attribute like this: src="./Assets/name.png". Pay attention to the point at the beginning of the route  :eyes:  
Screenshot from 2022-09-21 07-20-28

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <title>Portfolio Asha</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <div class="toolbar">
+        <h3 class="logo">My logo</h3>
+        <i class="fas fa-bars"></i>
+    </div>
```
Comment (Santiago220991) [inline index.html]: Kindly change the `div` tag to a `header` tag. Semantic tags make HTML more comprehensible by better defining the different sections and layouts of web pages. Also change the tags `h3` to an `a ` and `i` to a `nav`. This is for conventions and give the user a better experience. You can check this [link ](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page)for more info. :nerd_face:

File: index.html (hunk 1 above)
Comment (Santiago220991) [inline index.html]: Kindly remove these elements, decorations are always background images, not img tags. Please use CSS to add this as background. :recycle:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="Ellipse11"><img src="\Assets\Ellipse 11.png"></div>
+        <div class="Ellipse3"><img src="\Assets\Ellipse 3.png"></div>
+        <div class="Rectangle4"><img src="\Assets\Rectangle 4.png"></div>
+        <div class="Rectangle7"><img src="\Assets\Rectangle 7.png"></div>
+        <div class="Ellipse5"><img src="\Assets\Ellipse 5.png"></div>
+        <div class="Ellipse8"><img src="\Assets\Ellipse 8.png"></div>
+        <section>
+            <h2 class="Right Intro">Namaste!<br> I'm Asha</h2>
```
Comment (Santiago220991) [inline index.html]: kindly change the `h2` tag to an `h1` this helps the browser to find your page easyly.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="Right icons">            
+            <a href="https://twitter.com/asha_cep" target="_blank" class="media-link" rel="noopener"><i class='fab fa-twitter'></i></a> 
+            <a href="https://www.linkedin.com/in/asha-vijayan-67179a18" target="_blank" class="media-link" rel="noopener"><i class='fab fa-linkedin'></i></a>
+            <a href="https://medium.com/" target="_blank" class="media-link" rel="noopener"><i class='fab fa-medium-m'></i></a>
+            <a href="https://github.com/asharanjith" target="_blank" class="media-link" rel="noopener"><i class='fab fa-github'></i></a>
+            <a href="https://angel.co/asha-sethu-vijayan" target="_blank" class="media-link" rel="noopener"><i class='fab fa-angellist'></i></a>
+        </div>
+        <div class="Ellipse12"><img src="\Assets\Ellipse 12.png"></div>
+        <div class="Ellipse13"><img src="\Assets\Ellipse 13.png"></div>
+        <div class="Ellipse4"><img src="\Assets\Ellipse 4.png"></div>
```
Comment (Santiago220991) [inline index.html]: Kindly remove these elements, decorations are always background images, not img tags. Please use CSS to add this as background. :recycle:

File: index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        </section>
+        <div class="Right contact">
+            <p class="media-text">LET’S CONNECT</p>
+        </div>
+        <div class="Right icons">            
+            <a href="https://twitter.com/asha_cep" target="_blank" class="media-link" rel="noopener"><i class='fab fa-twitter'></i></a> 
+            <a href="https://www.linkedin.com/in/asha-vijayan-67179a18" target="_blank" class="media-link" rel="noopener"><i class='fab fa-linkedin'></i></a>
+            <a href="https://medium.com/" target="_blank" class="media-link" rel="noopener"><i class='fab fa-medium-m'></i></a>
+            <a href="https://github.com/asharanjith" target="_blank" class="media-link" rel="noopener"><i class='fab fa-github'></i></a>
+            <a href="https://angel.co/asha-sethu-vijayan" target="_blank" class="media-link" rel="noopener"><i class='fab fa-angellist'></i></a>
```
Comment (Santiago220991) [inline index.html]: Kndly wrap all these elements in a `ul` tag and every element inside of a `li` tag . This way will be semantically correct and the screen readers and other technologies will thank you. :smiley:

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @asharanjith , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (asharanjith) [reply index.html]: @Santiago220991 Hi, Thank you for the tips. It is so descriptive. So in the image path, I need to use the forward slash also right? Also for re-review do I need to submit the feedback form in the Dashboard?

**Review by mckent05 — APPROVED** [review]
Hi @asharanjith, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 

## Optional suggestions
**[OPTIONAL]** - Please try to make your mobile version responsive up until the desktop version. Doing this will ensure your site displays properly across different screen sizes. You may find this [article](https://www.w3schools.com/css/css3_mediaqueries.asp) useful in doing this. 👍 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by asharanjith** [comment]
Thank you.

**Pull request merged**